contractcourt/channel_arbitrator.go

```diff
@@ -2155,6 +2155,20 @@ func (c *ChannelArbitrator) checkRemoteDanglingActions(
 			continue
 		}
```

Comment:
Commit Msg: what do you mean by empty remote pending commitment ? A commitment without HTLCs ? Why was it send in the first place, was it a feeUpdate ?